.github/workflows/release.yaml

```diff
@@ -195,3 +195,15 @@ jobs:
              --certificate-oidc-issuer https://token.actions.githubusercontent.com \
              --certificate-identity-regexp '^https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/tags/v[0-9]+.[0-9]+.[0-9]+$' \
              $IMAGE@$DIGEST
+
+  homebrew:
+    needs: [ goreleaser ]
+    name: Bump Homebrew formula
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@b3327118b2153c82da63fd9cbf58942146ee99f0 # v3.1
+        with:
+          formula-name: openfga
+        env:
+          # token belongs to @miparnisari and expires on March 6, 2025.
```

Comment:
We need a SA for this. No personal tokens.

Reply:
but this probably can be done in another PR?

Reply:
I'll try to create a service account for OpenFGA so we can avoid using personal tokens. Hang tight...